src/Module.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class Module extends React.Component {
+  render() {
+    return (
+      <div key={this.props.module.title} className="semester">
+          <p id="sem01_mod01" className="button hover" >{this.props.module.title}</p>
```

Comment:
I think this was just left over from the original html. Since we are no longer rendering individual modules to specific module ids, I suppose we could get rid of it completely.